src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/VariableDeclarationUsageDistanceCheckTest.java

```diff
@@ -75,6 +75,7 @@ public void testGeneralLogic() throws Exception {
             "1001:9: " + getCheckMessage(MSG_KEY, "a", 3, 1),
             "1036:9: " + getCheckMessage(MSG_KEY, "c", 3, 1),
             "1066:9: " + getCheckMessage(MSG_KEY, "a", 4, 1),
+            "1075:9: " + getCheckMessage(MSG_KEY, "a", 2, 1),
```

Comment:
Please do not add new test cases in existing input files. We have a line limit of 120 but this file is already very big, so it is exempted. 

You can create a new input file for every case you are adding.